cluster-commands.go

```diff
@@ -359,6 +361,51 @@ func (adm *AdminClient) SRInternalReplicateBucketMeta(ctx context.Context, item
 	return nil
 }
 
+// SRBucketMetaInfo - returns all the bucket metadata available for bucket
+type SRBucketMetaInfo struct {
+	Bucket string          `json:"bucket"`
+	Policy json.RawMessage `json:"policy,omitempty"`
+
+	// Since tags does not have a json representation, we use its xml byte
+	// representation directly.
+	Tags *string `json:"tags,omitempty"`
+
+	// Since object lock does not have a json representation, we use its xml
+	// byte representation.
+	ObjectLockConfig *string `json:"objectLockConfig,omitempty"`
+
+	// Since SSE config does not have a json representation, we use its xml
+	// byte respresentation.
+	SSEConfig *string `json:"sseConfig,omitempty"`
+	// replication config in json representation
+	ReplicationConfig *string `json:"replicationConfig,omitempty"`
+}
+
+// SRInternalReplicateBucketMetaInfo - gets all bucket metadata from a peer cluster.
+func (adm *AdminClient) SRInternalReplicateBucketMetaInfo(ctx context.Context) (info SRBucketMetaInfo, err error) {
+	reqData := requestData{
+		relPath: adminAPIPrefix + "/site-replication/peer/bucket-meta",
```

Comment:
why do we need peer based calls here? - admin calls should be cluster wide not peer specific @poornas

Reply:
Okay then let's name it appropriately 

> SRInternalReplicateBucketMetaInfo 

to 

> SRRemoteReplicateBucketMetaInfo

Reply:
and also not call it "peer"